reference/compiling_for_osx.rst

```diff
@@ -22,19 +22,19 @@ Start a terminal, go to the root dir of the engine source code and type:
 
 ::
 
-    user@host:~/godot$ scons platform=osx
+    user@host:~/godot$ scons platform=osx bits=64 target=release_debug
```

Comment:
I'd prefer we don't specify these arguments, people can also compile with `bits=32` or `bits=fat` (the latter is used for official binaries), and `target=debug` if they want. Instead, a small comment could be added before `user@host:~/godot$ cp bin/godot.osx.opt.tools.64 Godot.app/Contents/MacOS/Godot` to say that the binary name may different depending on build options used with scons.